packages/web-ui/components/chat.tsx

```diff
@@ -201,12 +206,13 @@ export default function Chat({ roomId }: ChatProps) {
       data: createSocketMessageRequestFactory(roomId, value, userId ?? ''),
     });
   };
+
   return (
     <div className="flex h-full w-full flex-col">
       <ScrollArea ref={bottomEl} className="h-full">
         <div className="w-full px-4 pt-4">
           {messages.map((message, index) => {
```

Comment:
well done, just forgot to remove the `index`

Reply:
yup, done here 169548d6ca96e1254caff94dd27eda4cb7a7e302